com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfInertial.cs

```diff
@@ -249,12 +204,8 @@ public Link.Inertial ExportInertialData()
 
         private Link.Inertial.Inertia ExportInertiaData()
         {
-#if UNITY_2020_1_OR_NEWER
             ArticulationBody robotLink = GetComponent<ArticulationBody>();
-
-#else
-            Rigidbody robotLink = GetComponent<Rigidbody>();
-#endif
+            
             Matrix3x3 lamdaMatrix = new Matrix3x3(new[] {
```

Comment:
nitpick, but I think it's "lambda" not "lamda"?

Reply:
This is actually a good catch, I'll fix this up in a batch if Vidur requests more changes :)